app/dashboard/views.py

```diff
@@ -575,6 +576,7 @@ def profile(request, handle):
     params = {
         'title': 'Profile',
         'active': 'profile_details',
+        'newsletter_headline': "Be the first to know about new funded issues.",
```

Comment:
Just a nitpick note...  Please be mindful of maintaining consistency with usage of `'` and `"`

Reply:
Ah my bad !!

Reply:
No worries - Thanks for the contribution!